### PR TITLE
CPM management and geo fixes

### PR DESCRIPTION
**Pull request by nbuffon**
New features
-------------------

* rust: new 'CollisionRisk' DENM declaration
* rust: fix position computations for CPM's perceived objects
* rust: change visibility to public for few functions/mods
* rust: new utility functions in `PerceivedObject`
* rust: new dependency to `map_3d`

How to test
-----------------

1. Run the CopyCat example
    ```
    cargo run --bin its-client -- -H <broker_host> -P <broker_port> -u <username> -p <password> --mqtt-client-id <client_id>
    ```
    **=> stdout logs tells that the copycat receives messages (cam, denm, ...)**  

2. Check the applicative logs
    ```
    tail -F log/its-client_rCURRENT.log
    ```
    **=> Logs must tell about either copy/retain moving vehicle messages or stopped vehicle that are not copied**

### CODE REVIEW

File: rust/libits-client/src/reception/exchange/mobile_perceived_object.rs [hunk 1]
```diff
@@ -28,7 +28,9 @@ pub struct MobilePerceivedObject {
 }
 
 impl MobilePerceivedObject {
-    pub(crate) fn new(
+    // TODO FGA: check how to keep it private and manage end to end CPMs with natively
```
Comment (tigroo) [inline rust/libits-client/src/reception/exchange/mobile_perceived_object.rs:31]: noted ;)

File: rust/libits-client/src/reception/exchange/decentralized_environmental_notification_message.rs [hunk 2]
```diff
@@ -143,11 +143,13 @@ impl DecentralizedEnvironmentalNotificationMessage {
             sequence_number,
             etsi_timestamp,
             94,
-            Option::Some(0), // FIXME remove it when the gateway will accept it
+            Some(0), // FIXME remove it when the gateway will accept it
```
Comment (tigroo) [inline rust/libits-client/src/reception/exchange/decentralized_environmental_notification_message.rs:147]: let's remove the FIXME so

**Review by tigroo — APPROVED** [review]
Code reviewed, tests ok

File: rust/libits-client/src/reception/exchange/decentralized_environmental_notification_message.rs (hunk 2 above)
Comment (nbuffon) [reply rust/libits-client/src/reception/exchange/decentralized_environmental_notification_message.rs:147]: Good point

**Pull request merged**